sqlmesh/core/state_sync/engine_adapter.py

```diff
@@ -485,9 +485,7 @@ def _get_versions(self, lock_for_update: bool = False) -> Versions:
             return no_version
 
         return Versions(
-            schema_version=row[0],
-            sqlglot_version=row[1],
-            sqlmesh_version=seq_get(row, 2) or "0.0.0",
+            schema_version=row[0], sqlglot_version=row[1], sqlmesh_version=seq_get(row, 2)
```

Comment:
same with line 474?

Reply:
should we then apply defaults to sqlglot_version and schema_version too?

Reply:
sure

Reply:
updated